src/elements/ngm-cam-configuration.ts

```diff
@@ -103,6 +104,16 @@ export class NgmCamConfiguration extends LitElementI18n {
     },
   ];
 
+  constructor() {
+    super();
+    const observer = new MutationObserver(((mutationsList) => {
+      if (mutationsList[0].attributeName === 'hidden' && !this.hidden) {
```

Comment:
Why do we need an observer to be notified of a change of a property we control?

Reply:
updated